slmsuite/holography/toolbox/phase.py

```diff
@@ -168,16 +168,18 @@ def zernike(grid, n, m, aperture=None):
         Cartesian Zernike index defining the polynomial.
     aperture : {"circular", "elliptical", "cropped"} OR (float, float) OR None
         See :meth:`.zernike_sum()`.
+    return_mask : bool
+        Whether or not to return the 2D mask showing where Zernikes are computed.
```

Comment:
Zernike is probably getting a revamp soon, but I slightly modified the behavior here to conform with other parts of the package (now only returns the mask).